web/controllers/organization_controller.ex

```diff
@@ -0,0 +1,55 @@
+defmodule CodeCorps.OrganizationController do
+  use CodeCorps.Web, :controller
+
+  alias CodeCorps.Organization
+  alias JaSerializer.Params
+
+  plug :scrub_params, "data" when action in [:create, :update]
+
+  def index(conn, _params) do
+    organizations = Repo.all(Organization)
+    organizations = Repo.preload(organizations, [:slugged_route])
```

Comment:
I was on my phone last night doing this review, so thought I'd clarify now that I'm at my laptop.

I've done things like this in the past:

`models/organization.ex`

```elixir
defmodule CodeCorps.Organization do
  # Schema / changeset stuff
  ...
  
  # Query functions

  def with_slugged_route(query) do
    from o in query,
    preload: [:slugged_route]
  end
end
```

This is then consumable from the controller like this:

```elixir
organizations = Organization
|> with_slugged_route
|> Repo.all
```

Thoughts?

cc @begedin